src/types/project.types.ts

```diff
@@ -48,16 +48,23 @@ export interface ProjectContributorIdentity {
 
 export type ProjectContributorType = 'ProjectManager' | 'ProjectParticipant';
 
-export interface ProjectContributor {
+export interface ProjectContributorRole {
```

Comment:
```suggestion
interface ProjectContributorRole {
```
Flere ting som eksporteres, men som aldri importeres i denne fila. Jeg bruker [ts-prune](https://github.com/nadeesha/ts-prune) (de er i maintanence mode da, så kan være verd å se etter noe annet) for å finne sånne:
```
-> npm i -g ts-prune
-> ts-prune

\src\types\project.types.ts:51 - ProjectContributorRole (used in module)
\src\types\project.types.ts:60 - CreatorRole (used in module)
\src\types\project.types.ts:83 - EmptyProjectContributor
\src\types\project.types.ts:148 - emptyAffiliation (used in module)
```

Reply:
Bra tips! Jeg kan også bli flinkere å bruke showUsages når jeg lager dem / PR'en